Makefile

```diff
@@ -147,71 +147,88 @@ lr/docs/serve:
 .PHONY: lr/docs/markdown
 lr/docs/markdown: lr/build
 	./lr markdown resources/packs/aws/aws.lr \
-        --pack-name "Amazon Web Services (AWS)" \
+    --pack-name "Amazon Web Services (AWS)" \
+		--description "The Amazon Web Services (AWS) resource pack provides resources that let you query and assess the security of your Amazon Web Services (AWS) cloud services." \
```

Comment:
```suggestion
		--description "The Amazon Web Services (AWS) resource pack provides resources that let you query and assess the security of your AWS cloud services." \
```

Reply:
Wouldn't it be better to call it the `The MQL Amazon Web Services (AWS) resource pack`?

Reply:
That's a good question. I wanted to get "Mondoo" in there for SEO. Maybe we want both "Mondoo" and "MQL" in the description?

Reply:
That would work too. I just think mondoo is too generic

Reply:
So how about "The Mondoo Amazon Web Services (AWS) resource pack lets you query and assess the security of your AWS cloud services using MQL."
?

Reply:
This text is confusing to me. The resource pack defines MQL and is not using MQL

Reply:
"using MQL" is a modifier for how "you query and assess" not for "resource pack."

Reply:
But I think we're into the zone of over-writing and over-analyzing.  I'll take a fresh look tomorrow.

Reply:
Okay it's a fresh day and I still am happy with my most recent suggestion. But if it confused you then I want to make it better!  Does this help?

"The Mondoo Amazon Web Services (AWS) resource pack lets you use MQL to query and assess the security of your AWS cloud services."